docs/guides/user/collecting-cloud-metrics.md

```diff
@@ -0,0 +1,311 @@
+# Cloud Metrics
+
+Prometheus supports a wide range of exporters for getting metrics out of other systems and into Prometheus.
+We have recently added support for adding exporters for AWS CloudWatch and GCP Stackdriver directly into Opstrace.
```

Comment:
"We have recently added" is probably not relevant here, Towards shortening things, things like that can go : )

Reply:
Oh whoops, might have been left over from the blog post. Fixed